x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/action/user/TransportChangePasswordActionTests.java

```diff
@@ -48,7 +48,8 @@
 public class TransportChangePasswordActionTests extends ESTestCase {
 
     public void testAnonymousUser() {
-        final String hashingAlgorithm = randomFrom("pbkdf2", "pbkdf2_1000", "bcrypt", "bcrypt9");
+        final String hashingAlgorithm = inFipsJvm() ?
+            randomFrom("pbkdf2", "pbkdf2_1000") : randomFrom("pbkdf2", "pbkdf2_1000", "bcrypt", "bcrypt9");
```

Comment:
Not essential, but would it be worth putting the choice of hashing algorithm in its own utility method?